lib/searches.js

```diff
@@ -1,252 +1,343 @@
 let request = require("request-promise-native");
 let cheerio = require("cheerio");
 let validUrl = require("valid-url");
+var resUrl = require('url');
```

Comment:
I would name this the full `resolveURL`, I didn't know what "resURL" was until I found it in the code.